### PR TITLE
Harmonize propal index pages 

### PR DESCRIPTION
**Pull request by altairis-tof**
harmonize and change default number of lines display to speed up page loading and improve UX

### CODE REVIEW

File: htdocs/comm/index.php [hunk 1]
```diff
@@ -190,10 +191,13 @@
 				$i++;
 				$total += $obj->total_ht;
 			}
-			if ($total>0)
+			if ($num > $nbofloop)
 			{
-
-				print '<tr class="liste_total"><td>'.$langs->trans("Total").'</td><td colspan="2" class="right">'.price($total)."</td></tr>";
+				print '<tr class="liste_total"><td colspan="3" align="right">'.$langs->trans("XMoreLines", ($num - $nbofloop))."</td></tr>";
```
Comment (aspangaro) [inline htdocs/comm/index.php]: @altatof Use html5 class="right" instead of align="right"

Comment (altairis-tof) [reply htdocs/comm/index.php]: are you sure that class="center" will align the cell on the right ?

File: htdocs/comm/index.php [hunk 2]
```diff
@@ -270,9 +275,13 @@
...
             }
-            if ($total>0)
+			if ($num > $nbofloop)
+			{
+				print '<tr class="liste_total"><td colspan="3" align="right">'.$langs->trans("XMoreLines", ($num - $nbofloop))."</td></tr>";
+			}
+			elseif ($total>0)
             {
-                print '<tr class="liste_total"><td>'.$langs->trans("Total").'</td><td colspan="2" class="right">'.price($total)."</td></tr>";
+                print '<tr class="liste_total"><td align="right">'.$langs->trans("Total").'</td><td colspan="2" class="right">'.price($total)."</td></tr>";
```
Comment (eldy) [inline htdocs/comm/index.php]: Use of "align" attribute is deprecated with html5. Use instead class="right"

**Comment by altairis-tof** [comment]
done, you can merge; then i continue the job on other index pages

**Comment by altairis-tof** [comment]
@eldy thanks for merging; i see you put back 500 as default value for number of items displayed; dont you think it's pretty big for an index page ?

**Pull request merged**